cmd/launcher/cli/launcher.go

```diff
@@ -87,8 +87,8 @@ func verifyBuildpackAPIs(bps []launch.Buildpack) error {
 	for _, bp := range bps {
 		if bp.API == "" {
 			// If the same lifecycle is used for build and launcher we should never end up here
-			// but if for some reason we do, default to 0.2
-			bp.API = "0.2"
+			// but if for some reason we do, default to latest
+			bp.API = api.Buildpack.Latest().String()
```

Comment:
Same question here - seems a bit weird we end up with a buildpack that doesn't have an API. Is this in the spec?

Reply:
Yeah, buildpack API is here: https://github.com/buildpacks/spec/blob/main/platform.md#metadatatoml-toml

Reply:
Updated in fb806d31